qa/TL0_FW_iterators/test_jax.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+# used pip packages
+pip_packages='${python_test_runner_package} numpy jax'
+target_dir=./dali/test/python
+
+one_config_only=true
+
+do_once() {
+    NUM_GPUS=$(nvidia-smi -L | wc -l)
+}
+
+test_body() {
+    ${python_invoke_test} -m '(?:^|[\b_\./-])[Tt]est.*pytorch*' test_jax_integration.py
```

Comment:
```suggestion
    ${python_invoke_test} -m '(?:^|[\b_\./-])[Tt]est.*jax*' test_jax_integration.py
```
It was intentional or just a leftover?

Reply:
Nice, fixed.